src/app/psat/explore-opportunities/explore-opportunities-form/explore-opportunities-form.component.ts

```diff
@@ -82,6 +82,9 @@ export class ExploreOpportunitiesFormComponent implements OnInit {
   }
 
   initForms() {
+    if (!this.psat.inputs.isVFD && !this.psat.modifications[this.exploreModIndex].exploreOppsShowVfd) {
```

Comment:
Same as the comment from fsat. Test this with and installed version, select is vfd and then run this version. isVFD should be selected but I don't think it will be. This is probably the problem area.